modules/python/python-dest.c

```diff
@@ -226,7 +226,8 @@ _py_init_bindings(PythonDestDriver *self)
       msg_error("Error looking Python driver class",
                 evt_tag_str("driver", self->super.super.super.id),
                 evt_tag_str("class", self->class),
-                evt_tag_str("exception", _py_fetch_and_format_exception_text(buf, sizeof(buf))));
+                evt_tag_str("exception", _py_format_exception_text(buf, sizeof(buf))));
+      _py_finish_exception_handling();
```

Comment:
Can you wrap this (msg_error+finish) up into a macro or a va_arg function? I think this `_py_finish_exception_handling` is easy to miss, and this pattern is very frequent.

Reply:
It was originally part of _py_fetch_and_format_exception_text() and I explicitly extracted that, simply because then it was seemingly a log message that also happened to clear the error state.

Or you mean that I should wrap the msg_error() plus the cleanup? I'll try to see how that looks.

Reply:
Yes, I would be happy to see msg_error and  _py_finish_exception_handling() bundled together. Maybe into _py_msg_exception() or similar function/macro. Thanks.